Quiz-App/README.md

```diff
@@ -23,6 +24,8 @@ Users should be able to:
 
 ### Screenshot
 
+![Question Interface Card](./question-interface.png)
+![Results Card](./result.png)
```

Comment:
Please add a gif also.

Reply:
yes sure 👍

Reply:
hey, I created a gif, but it has a watermark of the editing software. does that work?

Reply:
> hey, I created a gif, but it has a watermark of the editing software. does that work?

There are couple of software out there that does not create watermark. You can use those

Reply:
okay